app/src/main/res/layout/fragment_suggested_edits_image_recommendation_item.xml

```diff
@@ -319,15 +318,14 @@
         android:elevation="14dp"
         tools:visibility="gone">
 
-        <View
-            android:id="@+id/publishBackgroundBaseView"
```

Comment:
@dbrant Did not see a use for this since there is already the 'publishBackgroundView' but let me know if I missed something..

Reply:
We should keep it in order to match the design needs.
#2281